config/default.json

```diff
@@ -13,5 +13,6 @@
 		"SECURE": false,
 		"EXPIRES_SECONDS": 2592000000
 	},
+	"COOKIE_SECRET": "ihasDF(=4jA4aF",
```

Comment:
Is this necessary when using the Configuration?

Reply:
Could be moved to default.schema.json. We agreed to leave it here to "force" to set it explicitly in the instances and not use the default, that is publicly readable by everybody cloning the repo.